_posts/2020-03-30-crunch-on-arm.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Protein Folding on Arm Devices | Helping with COVID-19 Research"
+author: Sahaj Sarup
+date: 2020-03-29 01:00:00+00:00
+image:
+    featured: true
+    path: /assets/images/blog/fold.png
+    name: fold.png
+categories: blog
+tags: 64-bit, 96Boards, aarch64, ARM, ARMv8, Consumer Edition, Hikey970, Linaro, Linux, arm64, protein folding, aerocore2
+---
+
+For the last couple of weeks I have been spending my spare cycles, along with a few other friends form the Arm Ecosystem, to get the power and efficiency of the aarch64 ISA in the hands of researchers and institutes that have been working tirelessly to make sense of the COVID-19 pandemic.
+
+# Protein folding and Its relation to curing COVID-19
+
+"Each protein exists as an unfolded polypeptide or random coil when translated from a sequence of mRNA to a linear chain of amino acids. This polypeptide lacks any stable (long-lasting) three-dimensional structure (the left hand side of the first figure). As the polypeptide chain is being synthesized by a ribosome, the linear chain begins to fold into its three-dimensional structure. Folding begins to occur even during translation of the polypeptide chain. Amino acids interact with each other to produce a well-defined three-dimensional structure, the folded protein (the right hand side of the figure), known as the native state. The resulting three-dimensional structure is determined by the amino acid sequence or primary structure.
```

Comment:
References to figure, might be worth adding the wikipedia figure with a reference to its original location as caption